TheMET/TheMET/ViewControllers/CatalogViewController.swift

```diff
@@ -129,15 +125,21 @@ class CatalogViewController: UIViewController {
                 completion(catalogCellData)
                 return
             }
-            let subtitle: String = ("\(objects.total) objects")
-//            localisation
+            let totalObjectsUInt: UInt = UInt(objects.total)
+            let subtitle: String = (self?.objectsCount(count: totalObjectsUInt))!
             self?.loadDepartmentImageURL(objectIds: objects.objectIDs, completion: { url in
                 let catalogCellData = CatalogCellData(catalogCellId: department.id, imageURL: url, title: department.displayName, subTitle: subtitle)
                 completion(catalogCellData)
             })
         }
     }
     
+    
+    private func objectsCount(count: UInt) -> String {
```

Comment:
1) It's better not to use `UInt` if not necessary cause most API works with `Int` type. Introducing `UInt` causes a lot of type-convertations.
2) naming: this method doesn't return objects count number. It return `String`. I suggest to rename it to
`cellSubtitle(objectsCount: Int) -> String`

Reply:
fixed